script/vsts/upload-artifacts.js

```diff
@@ -16,6 +16,10 @@ const REPO_OWNER = process.env.REPO_OWNER;
 const MAIN_REPO = process.env.MAIN_REPO;
 const NIGHTLY_RELEASE_REPO = process.env.NIGHTLY_RELEASE_REPO;
 
+// Compose an array of services to skip uploading to,
+// from environment variable SKIP_UPLOADING_TO (must be a comma-separated list)
+const skipUploadingTo = process.env.SKIP_UPLOADING_TO.split(/\s*(?:,|$)\s*/);
```

Comment:
Why not just `split(',')`?

Reply:
Allowing/trimming spaces isn't super important to me. I got that regex from here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split#Removing_spaces_from_a_string

Important thing to me was proving out the "one variable --> multiple array elements" approach.

Reply:
I don't think allowing spaces is worth it, even if that's what I'd want in, say a CLI program, it's totally unnecessary in CI infrastructure.